packages/f/firejail/files/0001-Support-a-stateless-configuration.patch

```diff
@@ -57,6 +57,47 @@ index 412cdd6..1f2037e 100644
     Spectre compiler patch: $HAVE_SPECTRE
  
  Features:
+diff --git a/src/firecfg/main.c b/src/firecfg/main.c
+index 07e3041..cbabb20 100644
+--- a/src/firecfg/main.c
++++ b/src/firecfg/main.c
+@@ -176,8 +176,12 @@ static void set_file(const char *name, const char *firejail_exec) {
+ // parse /etc/firejail/firecfg.config file
+ static void set_links_firecfg(void) {
+ 	char *cfgfile;
+-	if (asprintf(&cfgfile, "%s/firecfg.config", SYSCONFDIR) == -1)
++	if (access(SYSCONFDIR "/firejail.config", F_OK) == 0) {
++		cfgfile = SYSCONFDIR "/firejail.config";
++	} else {
++		if (asprintf(&cfgfile, "%s/defaults/firejail/firejail.config", DATADIR) == -1)
```

Comment:
I'm a little curious about the difference in creating the string here. Is there a reason to do it like this?

Reply:
No particular reason. I guess it could also be, for example

```c
if (asprintf(&cfgfile, "%s/defaults/firejail/firejail.config", SYSCONFDIR) == -1) {
	if (asprintf(&cfgfile, "%s/defaults/firejail/firejail.config", DATADIR) == -1)
		errExit("asprintf");
}
```

or something

Reply:
I think that would work, yeah.

Reply:
Okay, done (with some modifications because the above thing didn't actually quite do what I originally imagined).